djangoplicity/media/tasks.py

```diff
@@ -600,7 +600,7 @@ def upload_youtube(video_id, user_id=None):
 
     def mail_user(subject, body=''):
         if user and user.email:
-            send_mail(subject, body, 'no-reply@eso.org', [user.email])
+            send_mail(subject, body, 'no-reply@esahubble.org', [user.email])
```

Comment:
Nooo!!!!, don't update this to specific projects!, this is being used by multiple organizations, just update this to send mail to managers or send to admins!!

Reply:
That's right :( 
@diferpaho  This project is the core of other projects. Any update here will be reflected in those projects too.